src/nl/documentatie/10.woocommerce-v5.0.md

```diff
@@ -0,0 +1,329 @@
+
+#### Label formaat
+
+Selecteer het standaard label formaat.
+
+<hr/>
+<br/>
+
+### Douane
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_d_nl.jpeg" alt="woocommerce plugin customs" />
+
+Geef hier aan wat voor soort zendingen je buiten de EU verstuurt. In de regel zijn dit commerciële goederen.
+
+#### Inhoud pakket
+
+Selecteer de inhoud van jouw producten.
+
+#### HS-code
+
+Selecteer de douane code van jouw producten.
+
+#### Herkomstland
+
+Selecteer het land van herkomst van jouw producten
+
+<hr/>
+<br/>
+
+### Checkout
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e_nl.jpeg" alt="woocommerce plugin checkout" />
+
+#### Gebruik aparte adresvelden
+
+Als deze optie aan staat, worden de adresvelden opgesplitst in aparte velden.
+
+### Bezorgopties
+
+#### Bezorgopties tonen
+
+Dit tekst zal boven de bezorgopties verschijnen.
+
+#### Bezorgopties tonen voor backorders
+
+Als deze optie aan staat, worden de bezorgopties ook getoond als de producten niet op voorraad zijn
+
+#### Positie in checkout
+
+Positie van de bezorgopties
+
+#### Toegestane verzendmethoden
+
+Selecteer de verzendmethoden die je klanten mogen gebruiken voor de bezorgopties.
+
+#### Prijs type
+
+Of de prijs voor bezorging in de totaalprijs is inbegrepen.
+
+#### Bezorgopties titel
+
+Dit tekst zal boven de bezorgopties verschijnen.
+
+#### Custom stijlen (CSS)
+
+Custom CSS voor de bezorgopties
+
+<hr/>
+<br/>
+
+### Vervoerders
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_f_nl.jpeg" alt="woocommerce plugin labels" />
+
+Instellingen voor de beschikbare vervoerders.
+PostNL | DHL for you |
+
+::: expand PostNL
+Standaard exportinstellingen voor vervoerder PostNL
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e1_postnl.jpeg" alt="woocommerce MyParcel " />
+
+<hr/>
+<br/>
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e2_postnl.jpeg" alt="woocommerce MyParcel " />
+
+<hr/>
+<br/>
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e3_postnl.jpeg" alt="woocommerce MyParcel " />
+<hr/>
+<br/>
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e4_postnl.jpeg" alt="woocommerce MyParcel " />
+:::
+
+::: expand DHL
+for you (DHL Parcel connect, DHL Europlus)
+Standaard exportinstellingen voor vervoerder DHL for you
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e1_dhl.jpeg" alt="woocommerce MyParcel" />
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e2_dhl.jpeg" alt="woocommerce MyParcel" />
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e3_dhl.jpeg" alt="woocommerce MyParcel" />
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e3_dhl_2.jpeg" alt="woocommerce MyParcel" />
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e4_dhl.jpeg" alt="woocommerce MyParcel" />
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_e5_dhl.jpeg" alt="woocommerce MyParcel" />
+:::
+
+<hr/>
+<br/>
+
+<div class="p-5 rounded bg-monstera-500 text-white">
```

Comment:
wat een spannend kopje :)

Reply:
Eigenlijk om aan te geven dat je in een ander deel in de WooCommerce omgeving zit.